scripts/deploy.ts

```diff
@@ -1,14 +1,30 @@
 import { ethers } from 'hardhat'
+import * as fs from 'fs'
 
-async function deploy(contractName: string) {
-  const factory = await ethers.getContractFactory(contractName)
-  const contract = await factory.deploy()
-  await contract.deployed()
-  console.log(`${contractName} deployed to ${contract.address}`)
+export type DeployedAddresses = {
+  Manager: string
+  ExampleProjectTemplate: string
 }
 
 async function main() {
-  deploy('Manager')
+  const managerFactory = await ethers.getContractFactory('Manager')
+  const managerContract = await managerFactory.deploy()
+  await managerContract.deployed()
+  console.log(`Manager deployed to ${managerContract.address}`)
+
+  const exampleProjectTemplateFactory = await ethers.getContractFactory('ExampleProjectTemplate')
+  const exampleProjectTemplateContract = await exampleProjectTemplateFactory.deploy()
+  await exampleProjectTemplateContract.transferOwnership(managerContract.address)
+  await managerContract.addProjectTemplate(exampleProjectTemplateContract.address)
+  console.log(`Example template deployed to ${exampleProjectTemplateContract.address} and added to manager`)
+
+  // Store addresses into a file the app can use
+  const addresses: DeployedAddresses = {
+    Manager: managerContract.address,
+    ExampleProjectTemplate: exampleProjectTemplateContract.address,
+  }
+
+  fs.writeFileSync('./src/addresses.json', JSON.stringify(addresses, null, 2), 'utf8')
```

Comment:
now we store the addresses in a file (imported by the app), no need to manual copy paste of the addresses anymore